server/src/lib.rs

```diff
@@ -22,16 +22,21 @@
 use std::io;
 
 use anyhow::Result;
-use axum::routing::get;
+use axum::routing::{get, post};
 use axum::Router;
 use tokio::net::TcpListener;
 
 use crate::routes::home;
-use crate::xml::storage::touch_storage;
+use crate::routes::register_user::register_user;
+use crate::xml::storage::Storage;
 
+mod objects;
 pub mod report;
 mod routes;
 mod xml;
+#[allow(unused_imports)]
+#[macro_use]
+extern crate hamcrest;
```

Comment:
better to move it to `mod tests`

Reply:
@l3r8yJ for some reason I can't move it to the tests, it's only allowed in `lib.rs` or `main.rs`